torch_xla/csrc/ops/convolution_backward_overrideable.cpp

```diff
@@ -0,0 +1,94 @@
+#include "torch_xla/csrc/ops/convolution_backward_overrideable.h"
+
+#include "absl/strings/str_join.h"
+#include "tensorflow/compiler/xla/xla_client/debug_macros.h"
+#include "tensorflow/compiler/xla/xla_client/util.h"
+#include "torch_xla/csrc/convolution.h"
+#include "torch_xla/csrc/lowering_context.h"
+#include "torch_xla/csrc/ops/infer_output_shape.h"
+
+namespace torch_xla {
+namespace ir {
+namespace ops {
+namespace {
+
+xla::Shape NodeOutputShape(
+    const Value& grad_output, const Value& input, const Value& weight,
+    tensorflow::gtl::ArraySlice<const xla::int64> stride,
+    tensorflow::gtl::ArraySlice<const xla::int64> padding,
+    tensorflow::gtl::ArraySlice<const xla::int64> dilation, bool transposed,
+    tensorflow::gtl::ArraySlice<const xla::int64> output_padding,
+    const xla::int64 groups) {
+  auto lower_for_shape_fn =
+      [stride, padding, dilation, transposed, output_padding,
+       groups](tensorflow::gtl::ArraySlice<const xla::XlaOp> operands)
+      -> xla::XlaOp {
+    XLA_CHECK_EQ(operands.size(), 3)
+        << "Unexpected number of operands: " << operands.size();
```

Comment:
You can let the extra message loose. The message printed by `XLA_CHECK_EQ(operands.size(), 3)` will pretty explicit already.